play.py

```diff
@@ -53,12 +53,17 @@ def take_guess(force_upper: bool = True) -> str:
     )
 
     # Play game forever
+    game_number = 1
     while True:
         word = random.choice(list_of_words)
+        attempt = 1
         guessed = False
 
-        print("Guess the new word: ")
+        print("[Game {}] Guess the new word: ".format(game_number))
         while not guessed:
             guess = take_guess()
             output, guessed = determine_word_positions(word, guess)
-            print(output)
+            print("[Attempt {}]: {}".format(attempt,output))
```

Comment:
Lets add a space here between the two arguments in case this project ever gets a linter